### DISCUSSION TITLE
Icon Ghidra

### DISCUSSION THREAD

**Discussion by epsilon0402** [NationalSecurityAgency/ghidra#3744] [Q&A]
Hello everybody, I run Ghidra but it doesn't have the icon on the dock. I use Ubuntu 20.04. How to fix it, thank you. 
 [Screenshot image](https://i.ibb.co/jDmFrdN/Screenshot-from-2021-12-11-17-45-20.png)

**Comment by dev747368** [comment]
This is a known issue with Ubuntu's dock and java.

See issue #463
